dbt/include/snowflake/macros/materializations/incremental.sql

```diff
@@ -58,12 +58,21 @@
 
 {% materialization incremental, adapter='snowflake', supported_languages=['sql', 'python'] -%}
 
-  {% set original_query_tag = set_query_tag() %}
```

Comment:
Hi @VersusFacit, I noticed that the query tags for incremental models are not applied anymore.
Why did you remove this line?